### ISSUE TITLE
"Fatal: Failed to write EVM genesis state: database contains incompatible genesis" after running previous fakenet

### ISSUE THREAD

**Issue by MKong** [Fantom-foundation/go-lachesis#361]
**Describe the bug**
After running a fakenet (e.g `lachesis --fakenet 1/1`), stopping it, and then running another fakeset (e.g `lachesis --fakenet 1/5), I get the following error:

"Fatal: Failed to write EVM genesis state: database contains incompatible genesis"

**To Reproduce**
1. Clone scope2
2. Run `go build -o ./build/lachesis ./cmd`
3. `cd build`
4. `lachesis --fakenet 1/1`
5. Stop lachesis
6. `lachesis --fakenet 1/5`

**Expected behavior**
I should be able to start a new network, overriding the previous blocks / genesis file.

**Desktop:**
 - OS: Mojave 10.14.6

**Comment by devintegral3** [comment]
With different fakenet settings, you get different genesis balances and hence different genesis blocks, i.e. completely different networks, so the behavior is expected. We think that it's dangerous to automatically drop the previous data, even in fakenet.
If you want to work with different fakenets from time to time, then you may specify different datadirs for them:
```
lachesis --fakenet 1/1 --datadir ~/.lachesis/fakenet.1.1
lachesis --fakenet 15 --datadir ~/.lachesis/fakenet.1.5
```

**Comment by quan8** [comment]
For the failure due to attempt to overwrite an EVM genesis state, it's fine, as it can protect existing db.

However, on the second launch in step 6, it'd be good that it will check and output if some conditions do not meet (such as the existence of db, paths, etc).

**Comment by MKong** [comment]
Agreed with Quan.

Kind regards,
Michael Kong
0422 775 856

**Comment by devintegral** [comment]
Will be fixed with #368 , see https://github.com/sfxdxdev/go-lachesis/commit/79a3b45a7aff7cee407f2e89618a56d3c9f419d4#diff-fa9214965f3ca5286d7cd213bbc4d35b
